db/sql.h

```diff
@@ -1123,8 +1123,7 @@ extern int gbl_master_swing_osql_verbose;
 extern int gbl_master_swing_sock_restart_sleep;
 
 /* takes care of both stat1 and stat2 */
-#define is_sqlite_stat(x)                                                      \
-    strncmp((x), "sqlite_stat", sizeof("sqlite_stat") - 1) == 0
+#define is_sqlite_stat(x) (strncmp((x), "sqlite_stat", strlen("sqlite_stat")) == 0)
```

Comment:
some compilers may not be able to optimize strlen away.

Reply:
ok, I put that part back. thanks